README.md

```diff
@@ -155,15 +155,26 @@ Container can be built via running `make docker-build`.
 
 ## Commitment Schemas
 
-Commitments returned from the EigenDA Proxy adhere to the following byte encoding:
+### Optimism Commitment Mode
+For `alt-da` clients running on Optimism, the following commitment schema is supported:
 
 ```
- 0        1        2         3               N
- |--------|--------|---------|---------------|
-  commit   da layer  version  raw commitment
+ 0        1        2         3                N
+ |--------|--------|---------|----------------|
+  commit   da layer  version   raw commitment
```

Comment:
so what exactly are the 2 first bytes? And why do they not appear in https://specs.optimism.io/experimental/alt-da.html#input-commitment-submission

<img width="768" alt="image" src="https://github.com/user-attachments/assets/73f5a28c-09e9-4a4f-b81a-0ffc87d5374d">